src/WebJobs.Extensions/Extensions/Warmup/WarmupWebJobsBuilderExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Warmup
```

Comment:
Builder extensions should be in Microsoft.Extensions.Hosting namespace for discoverability. See other extensions